packages/daimo-api/src/network/uniswapClient.ts

```diff
@@ -81,47 +81,52 @@ export class UniswapClient {
   }
 
   swapCacheKey(
-    addr: Address,
+    toAddr: Address,
     fromAmount: BigIntStr,
-    token: "ETH" | Address,
+    fromToken: "ETH" | Address,
+    toToken: "ETH" | Address,
     receivedAt: number,
     fromAddr: Address
   ) {
-    return `${addr}-${fromAmount}-${token}-${receivedAt}-${fromAddr}`;
+    return `${toAddr}-${fromAmount}-${fromToken}-${receivedAt}-${fromAddr}-${toToken}`;
```

Comment:
this is getting a bit gnarly. consider eg. `JSON.stringify({fromAddr, fromAmount, fromToken, receivedAt, toAddr, toToken})`